materialize/materialize.proto

```diff
@@ -54,8 +54,8 @@ service Driver {
   //    - The Flow runtime client sends TransactionRequest.Open,
   //      opening the stream and requesting it be fenced from other streams.
   // :Opened:
-  //    - The driver server sends TransactionResponse.Opened after,
-  //      where supported, ensuring other stream clients are fenced.
+  //    - The driver server sends TransactionResponse.Opened after ensuring
+  //      that other raced clients are fenced (if fencing is supported).
```

Comment:
I think this is probably a bit tangential, but what exactly do you mean by "fencing" in this context? I'm not familiar with this terminology.

Reply:
Terminology comes from [memory barriers](https://en.wikipedia.org/wiki/Memory_barrier). A fence is an operation which inserts a barrier such that a (distributed) actor F is assured no other actor Z (often called a "zombie") is able to race a modification of a mutually shared resource. F has "fenced" the resource from Z.